### PR TITLE
Minimap edit styling

### PR DESCRIPTION
**Pull request by dreeves4321**
A few changes to make the minimap edit mode easier to understand. There is one remaining awkward problem I can't figure out how to address: the blue glowing focus state around the edit button. I cannot find where that blue glow is applied. So it remains...

_Pull requests into Flux require the following. Submitter should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable (**N/A**) and :white_check_mark:. Reviewers should complete every item on the bulleted list when reviewing._

## Submitter:

**Running the Application**

- [x] Manually tested in Chrome (primary browser for testing)
- [ ] Manually tested in IE/Safari (verify app loads and basic feature testing)
- [ ] Manually tested in Chrome in iPad mode (at minimum, viewing patient record)

**Task**

- [ ] Branch implements task as expected
- [ ] Key application features are updated as needed for task and tested (see: [Key Application Features](https://github.com/FluxNotes/flux/wiki/Key-Application-Features))
- [ ] Task specific definition of done (as described in the Jira task) is met, if applicable.

**Documentation**

- [ ] This pull request describes why these changes were made
- [ ] Recognizes any potential shortcomings/bugs in the description 
- [ ] Pre release script is updated on the [Pre Release Testing Script wiki page](https://github.com/FluxNotes/flux/wiki/Pre-Release-Testing-Script) (add/remove single features to test, do not run through full script)
- [ ] Update the list of key features with new features on the [Key Application Features wiki page](https://github.com/FluxNotes/flux/wiki/Key-Application-Features)
- [ ] Document any new APIs/extension points
- [ ] Additional technical components and libraries are documented and added to [wiki](https://github.com/FluxNotes/flux/wiki/About-Flux-Notes) as needed

**Code Quality**

- [ ] Code style guide is followed (see: [Code Style Guide](https://github.com/FluxNotes/flux/wiki/JavaScript-and-HTML-Code-Style-Guide))
- [ ] Code is commented

**Tests**

- [ ] Existing tests passed
- [ ] Added backend tests for new functionality added

## Reviewers:

- Check that code is maintainable and reusable. Code reuses existing code and infrastructure where appropriate.
- Check the PR and code accomplishes the task's purpose.
- Check that new tests appropriately test the new code, including edge cases. Other existing tests pass.
- Verify Pull Request checklist is correctly completed by submitter.
- Try to break the code

### CODE REVIEW

File: src/lib/react-minimap/components/Child.js [hunk 1]
```diff
@@ -75,7 +75,11 @@ export default class Child extends Component {
             displayTitle = shortTitle;
         }
 
-        const className = (this.props.inEditMode && !this.state.visible) ? 'shaded' : '';
+        var className = '';
```
Comment (mgramigna) [inline src/lib/react-minimap/components/Child.js]: Should use `let` instead of `var`

File: src/lib/react-minimap/components/Child.js [hunk 2]
```diff
@@ -75,7 +75,11 @@ export default class Child extends Component {
...
+        if (this.props.inEditMode) {
+            className = this.state.visible ? 'edit_visible' : 'edit_invisible';
+        }
+        //const className = (this.props.inEditMode && !this.state.visible) ? 'shaded' : '';
```
Comment (mgramigna) [inline src/lib/react-minimap/components/Child.js]: Can remove commented out code if not using anymore

File: src/lib/react-minimap/react-minimap.scss [hunk 3]
```diff
@@ -42,6 +42,17 @@
     color: $interface-blue-text
 }
 
-.shaded {
-	background: rgba(0, 0, 0, 0.1) !important;
+//.shaded {
```
Comment (mgramigna) [inline src/lib/react-minimap/react-minimap.scss]: Can remove commented out code if not using anymore

File: src/lib/react-minimap/react-minimap.scss [hunk 4]
```diff
@@ -42,6 +42,17 @@
...
+//}
+
+.edit_visible {
+    border: 1px solid rgba(2, 141, 234, .4) !important;
+    color: $body-black !important;
+}
+
+.edit_invisible {
+    border: none !important;
+    background-color: #F9F9F9 !important;
```
Comment (mgramigna) [inline src/lib/react-minimap/react-minimap.scss]: #F9F9F9 is in our `_variables.scss`, so we can replace this hex color with `$contrast-gray`

**Comment by dreeves4321** [comment]
Thanks Matt. Now... how do I accept those requested changes?

**Comment by mgramigna** [comment]
@dreeves4321 I think once you push up the changes that I requested we can mark the changes as resolved and then I will approve :)

**Comment by dreeves4321** [comment]
Right, I got confused with an inline "suggestion". You're just commenting on code.

**Comment by dreeves4321** [comment]
@mgramigna I think I got all those in!

File: src/lib/react-minimap/components/Child.js [hunk 5]
```diff
@@ -75,7 +75,10 @@ export default class Child extends Component {
             displayTitle = shortTitle;
         }
 
-        const className = (this.props.inEditMode && !this.state.visible) ? 'shaded' : '';
+        var className = '';
```
Comment (Dtphelan1) [inline src/lib/react-minimap/components/Child.js]: As a matter of practice, we try not to use `var` in any code we add toexisting tools. Could we change this to a let?

**Pull request merged**